Week1/example.html

```diff
@@ -0,0 +1,122 @@
+<html lang="en" class="gr__hyf-github_netlify_com">
```

Comment:
This file should not be part of a PR. You should remove it from your local repository.